### PR TITLE
Added nginx to front alcali

### PR DESCRIPTION
**Pull request by shaidar**
#### What's this PR do?
Adds Nginx as a reverse proxy for alcali as the default install uses gunicorn as a web server.

### CODE REVIEW

File: pillar/nginx/alcali.sls [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              - ssl_session_tickets: 'off'
+              - ssl_protocols: 'TLSv1.2 TLSv1.3'
+              - ssl_ciphers: "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256\
+                  :DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384\
+                  :ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-SHA256\
+                  :ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256"
+              - ssl_prefer_server_ciphers: 'on'
+              - resolver: 1.1.1.1
+              - location /:
+                  - proxy_pass: http://127.0.0.1:8000
```
Comment (blarghmatey) [inline pillar/nginx/alcali.sls:42]: If it's using uwsgi should we be using the uwsgi_pass directive?

File: pillar/nginx/alcali.sls [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                  - proxy_http_version: 1.1
+                  - proxy_set_header: 'Connection "upgrade"'
```
Comment (blarghmatey) [inline pillar/nginx/alcali.sls:44]: Is it using websockets, or is this copy pasta? :spaghetti:

Comment (shaidar) [reply pillar/nginx/alcali.sls:44]: Well it's using websockets to connect to the salt-master and not for us connecting to it, so the answer is no.

**Comment by blarghmatey** [comment]
:+1:

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**